src/devices/cpu/hcd62121/hcd62121.cpp

```diff
@@ -1761,18 +2002,19 @@ void hcd62121_cpu_device::execute_run()
 		case 0xF1:      /* unk_F1 reg/i8 (out?) */
 		case 0xF3:      /* unk_F3 reg/i8 (out?) */
 		case 0xF5:      /* unk_F5 reg/i8 (out?) */
+		case 0xF6:      /* unk_F6 reg/i8 (out?) */
 		case 0xF7:      /* timer_ctrl i8 */
 			logerror("%02x:%04x: unimplemented instruction %02x encountered\n", m_cseg, m_ip-1, op);
 			read_op();
 			break;
 
 		case 0xFC:      /* unk_FC - disable interrupts/stop timer?? */
-		case 0xFD:      /* unk_FD */
 			logerror("%02x:%04x: unimplemented instruction %02x encountered\n", m_cseg, m_ip-1, op);
 			break;
 
+		case 0xFD:      /* timer_wait_low (no X1 clock, address or data bus activity) */
 		case 0xFE:      /* timer_wait */
-			if (m_time & 0x01) {
+			if (m_time_op & 0x01) {
```

Comment:
While you’re working on it, can you reformat this to Allman style as well?  I think you added it as K&R last time.